backend/scripts/chat_feedback_dump.py

```diff
@@ -0,0 +1,239 @@
+# This file is used to demonstrate how to use the backend APIs directly
+# In this case, the equivalent of asking a question in Danswer Chat in a new chat session
```

Comment:
Is this accurate? Might be worth adding context about the feedback, assistants use case, etc?

Reply:
oops, that was a leftover comment from the script i started with.